src/components/EnumSwitch.ts

```diff
@@ -0,0 +1,141 @@
+import { Component, ReactElement, createElement } from "react";
+import { Alert } from "./Alert";
+import { BootstrapStyle, EnumButton, SwitchStatus } from "./EnumButton";
+import { EnumSpan, EnumSpanProps } from "./EnumSpan";
+import * as classNames from "classnames";
+import "../ui/EnumSwitch.scss";
```

Comment:
The whole header is not as per our guidelines. In our guidelines.
-Import react
-Other Import import libraries
-Your typings
-Your components
-The CSS classes
```
 import { Component, ReactElement, createElement } from "react";
 import * as classNames from "classnames";

 import { Alert } from "./Alert";
 import { BootstrapStyle, EnumButton, SwitchStatus } from "./EnumButton";
 import { EnumSpan, EnumSpanProps } from "./EnumSpan";

 import "../ui/EnumSwitch.scss";
```

Reply:
changed